brigade.js

```diff
@@ -81,8 +84,8 @@ function xbuild(e, p) {
   return goBuild;
 }
 
-function test(e, p) {
-  var goTest = new GoJob(`${projectName}-test`);
+function testUnit(e, p) {
+  var goTest = new GoJob(`${projectName}-testUnit`);
```

Comment:
re: the question posed on this PR on why CI seemed to stop working w/ these changes...  unfortunately, it's due to k8s resource [naming restrictions](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names) which only allow for _lowercase_ letters (we use the provided job name to construct the k8s Pod name).  

This isn't obvious and we didn't have a comment in this file mentioning this.  We could either add comment(s) around this _or_ add logic to lowercase all provided job names in our `GoJob` class constructor, etc.

Reply:
(meaning, without further changes here, we'd need to rename the new jobs to `${projectName}-testunit`, `${projectName}-testintegration` and`${projectName}-testcli`, thus eliminating the problematic uppercase letters.)